frontend/app/(no-footer)/organiser/event/event-collection/[id]/page.tsx

```
@@ -392,16 +465,114 @@ export default function CollectionPage({ params }: CollectionPageProps) {
+                        location={event.location}
+                        price={event.price}
+                        vacancy={event.vacancy}
+                        loading={false}
+                        openInNewTab={true}
+                      />
+                    </div>
+                    <button
+                      onClick={() => handleRemoveEvent(event.eventId)}
+                      className="absolute bottom-2 right-2 p-2 bg-red-600 text-white rounded-full hover:bg-red-700 shadow-lg z-10 cursor-pointer transition-all duration-200 opacity-100 md:opacity-0 group-hover:opacity-100"
+                      title="Remove from collection"
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)